bitnami/postgresql/files/certs/docker-entrypoint-initdb.d/README.md

```diff
@@ -0,0 +1 @@
+You can copy here your custom `.crt`, `.key` or `.crl` files to easily create a secret out of them.
```

Comment:
AFAIK we decided not to include these initdb.d folders anymore, as they were confusing to users and required the chart source or repo to be installed on the file system.

Reply:
👍 had no idea, removing this option